src/CoreBundle/private/library/classes/TCMSListManager/TCMSListManagerModuleInstanceEndPoint.class.php

```diff
@@ -152,17 +152,32 @@ public function CallBackTemplateEngineInstancePages($id, $row)
 
         $oCmsTplPageList = TdbCmsTplPageList::GetList($query);
         while ($oCmsTplPage = $oCmsTplPageList->Next()) {
-            $sPath = $oCmsTplPage->fieldTreePathSearchString;
-            if (empty($sPath)) {
-                $sPath = TGlobal::Translate('chameleon_system_core.list_module_instance.no_usages_found_in_navigation_node');
+            $path = $this->getBreadcrumbsFromPaths($oCmsTplPage->fieldTreePathSearchString);
+            if (empty($path)) {
+                $path = TGlobal::Translate('chameleon_system_core.list_module_instance.no_usages_found_in_navigation_node');
             }
-            $pageString .= '<div style="white-space: nowrap;"><h2 style="margin: 0px 0px 5px 0px;">'.TGlobal::OutHTML($oCmsTplPage->fieldName).' (ID '.TGlobal::OutHTML($oCmsTplPage->id).'):</h2>
-        '.$sPath.'</div>';
+            $pageString .= '<div class="font-weight-bold">'.TGlobal::OutHTML($oCmsTplPage->fieldName).' (ID '.TGlobal::OutHTML($oCmsTplPage->id).'):</div>
+            <div>'.$path.'</div>';
         }
 
         return $pageString;
     }
 
+    private function getBreadcrumbsFromPaths($paths): string
+    {
+        $renderedPaths = '';
+        $pathElementList = explode(' ', $paths);
+        foreach($pathElementList as $path) {
+            if ('/' === substr($paths,0,1)) {
```

Comment:
Can be simplified using ltrim.

Should also be $path instead of $paths I think.

Reply:
right, done